drivers/google/provider.go

```diff
@@ -74,6 +75,9 @@ func New(opts ...Option) (autoscaler.Provider, error) {
 	if p.network == "" {
 		p.network = "global/networks/default"
 	}
+	if p.subnetwork == "" {
+		p.subnetwork = "subnetworks/default"
+	}
```

Comment:
is setting a default value required?  is this a breaking change for people are using the autoscaler with google compute? 

if this has potential to break existing installations we should consider removing this code.